packages/kit/src/runtime/server/page/load_node.js

```diff
@@ -94,6 +94,13 @@ export async function load_node({
 
 				opts.headers = new Headers(opts.headers);
 
+				// merge headers from request
+				for (const [key, value] of event.request.headers) {
+					if (opts.headers.has(key)) continue;
+					if (key === 'cookie' || key === 'authorization') continue;
```

Comment:
were we only going to skip these for the `omit` case? right now it looks like it's always skipping them

Reply:
We're skipping copying them over directly right here. These headers should already have gotten handled earlier, if it was appropriate to copy them over.

Reply:
I seem to be getting issues from version 1.0.0-next.253 upwards whenever I make server-side fetch requests with an "authorization" header. I'm guessing it may relate to this code change?

See my comment here: https://github.com/sveltejs/kit/pull/3631#issuecomment-1065196387